### PR TITLE
Updated Harry Potter references from search help to Sam Wilson

### PR DESCRIPTION
**Pull request by melinath**
# Pull Request Checklist

* [x] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [x] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [x] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* ~[ ] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)~ N/A
* [x] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

N/A - I believe this falls under the "spelling corrections and documentation improvements" clause.

## Purpose
Updates the help text for bookmark, work, and tag searching to use Sam Wilson (and Bucky Barnes) as examples instead of Harry Potter (and Gilderoy Lockehart).

I believe Harry Potter was likely the most popular fandom on AO3 when the archive was first created; however, over time that has [switched to the MCU](https://www.reddit.com/r/AO3/comments/v9h4ae/analysis_of_the_top_fandoms_on_ao3_percentages/).

I also corrected an HTML tag: `spam` -> `samp`

## Testing Instructions

Go to the bookmark, work, and tag search pages and check whether the new text is displayed.

## References

N/A

## Credit

What name and pronouns should we use to credit you in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?

If you have a Jira account, please include the same name in the "Full name"
field on your Jira profile, so we can assign you the issues you're working on.

*Please note that if you do not fill in this section, we will use your GitHub account name and
they/them pronouns.*

Stephen Lewis, he/him

### CODE REVIEW

File: public/help/tag-search-text-help.html [hunk 1]
```diff
@@ -13,31 +13,31 @@ <h4>Tag Search Text</h4>
...
+    Wilson</samp>, and <samp>Wilson</samp>.
   </dd>
 
   <dt>
     ": words in exact sequence
   </dt>
   <dd>
-    <kbd>"Harry Lockhart"</kbd> will find <samp>"Harry Lockhart"</samp> but not
-    <samp>Harry Potter/Gilderoy Lockhart</samp>.
+    <kbd>"Sam Barnes"</kbd> will find <samp>Sam Barnes</samp> but not
```
Comment (melinath) [inline public/help/tag-search-text-help.html:32]: I removed the quotation marks here for consistency with the other help text.

**Comment by TheCarpetMerchant** [comment]
While the reasoning seems sound, I think Harry Potter is better because more people outside of the Harry Potter fandom know who Harry is, then people outside the MCU fandom know who Bucky is.

But Lockhart could easily be changed for a more well-known character like Dumbledore for the same reason though.

**Comment by melinath** [comment]
Hm! I see your point - Marvel does have a problem where the characters have superhero names in addition to their legal names, which means folks outside of the fandom might not recognize Sam Wilson (or even Steve Rogers).

In terms of recognizability, I think there are a number of compelling alternatives to Harry Potter - for example, Sherlock Holmes, Frodo Baggins, Leia Organa, James Kirk. Although those are smaller fandoms on AO3, their longevity and continued adaptation over time mean they are likely very recognizable to people in general. How would you feel about one of those?

**Comment by sarken** [comment]
Hi, Stephen!

Thanks for this, but could you revert the fandom change and just keep the HTML typo fix? While we're happy to accept changes to our GitHub documentation and code comments, the text of the Archive's help files is the purview of the AO3 Documentation committee.

**Comment by melinath** [comment]
okay, thanks for being clear. I'll just close the PR in that case.

**Pull request closed without merging**